src/util/fs_helpers.cpp

```diff
@@ -298,3 +303,15 @@ std::optional<fs::perms> InterpretPermString(const std::string& s)
         return std::nullopt;
     }
 }
+
+#ifdef __APPLE__
+FSType GetFilesystemType(const fs::path& path)
+{
+    if (struct statfs fs_info; statfs(path.c_str(), &fs_info)) {
+        return FSType::ERROR;
+    } else if (std::string_view{fs_info.f_fstypename} == "exfat") {
```

Comment:
Few references I found that confirm this is the correct value:
* https://github.com/KDE/kcoreaddons/blob/master/src/lib/io/kfilesystemtype.cpp#L38
* https://docs.getlogga.com/building-blocks/access-logs#example-mount-payloads
* https://github.com/rfjakob/gocryptfs/blob/master/internal/syscallcompat/quirks_darwin.go#L12